apis/config/v1beta1/configuration_types.go

```diff
@@ -273,8 +273,26 @@ type MultiKueue struct {
 	// Defaults to 15 minutes.
 	// +optional
 	WorkerLostTimeout *metav1.Duration `json:"workerLostTimeout,omitempty"`
+
+	// DispatcherName defines the dispatcher responsible for selecting worker clusters to handle the workload.
+	// - If specified, the workload will be handled by the named dispatcher.
+	// - If not specified, the workload will be handled by the default (MultiKueueDispatcherModeAllClusters) dispatcher.
+	DispatcherName *string `json:"dispatcherName,omitempty"`
+
+	// DispatcherRoundTimeout specifies the duration given to a selected workers to attempt to reserve the workload's required resources.
```

Comment:
I was under impression that we decided not to include round concept in the generic in top leve MK config. If this is truly needed please group it under an optional SequentialDispatcher config or something like that.